invenio/modules/deposit/static/js/deposit/form.js

```diff
@@ -597,8 +607,22 @@ define(function(require, exports, module) {
           }
   }
 
+  /**
+   * Fired when text is pasted into an input
+   *
+   * @event paste
+   * @param event {Event}
+   */
+  this.onFieldPasted = function(event) {
+    var that=this;
+    /* timeout allows the pasted content to be available */
+    setTimeout(function () {
+      that.onFieldChanged(event);
+    }, 0);
+  }
+
   this.onCheckboxChanged = function (event) {
-    if(event.target.name.indexOf('__input__') == -1){
+    if(event.target.name.indexOf('__input__') == -1 && event.target.name ){
```

Comment:
can you follow same pattern in the file `if<space>(<nospace>...<nospace>)<space>{` ?